assets/js/modules/analytics-4/datastore/audience-settings.test.js

```diff
@@ -0,0 +1,244 @@
+/**
+ * `modules/analytics-4` data store: audience settings tests.
+ *
+ * Site Kit by Google, Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { createRegistry } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { untilResolved } from '../../../../../tests/js/utils';
+import { MODULES_ANALYTICS_4 } from './constants';
+import { availableAudiences as availableAudiencesFixture } from './__fixtures__';
+import { CORE_USER } from '../../../googlesitekit/datastore/user/constants';
+
+/**
+ * To-Do: After the implementation of #8888, we can remove these individual imports
```

Comment:
Please use `TODO` for consistency:

```suggestion
 * TODO: After the implementation of #8888, we can remove these individual imports
```